hack/make-rules/verify.sh

```diff
@@ -52,12 +52,12 @@ if [[ ${EXCLUDE_TYPECHECK:-} =~ ^[yY]$ ]]; then
     )
 fi
 
-
-# Exclude vendor checks in certain cases, if they're running in a separate job.
-if [[ ${EXCLUDE_GODEP:-} =~ ^[yY]$ ]]; then
+# Exclude dependency checks in certain cases, if they're running in a separate job.
+if [[ ${EXCLUDE_DEPENDENCIES:-} =~ ^[yY]$ ]]; then
```

Comment:
We can't change the variable name here, unless we update it throughout test-infra (and we would need to pick it backwards)

Reply:
@cblecker -- reverted that change.